cmd/fs-v1.go

```diff
@@ -749,14 +749,21 @@ func (fs fsObjects) getObjectETag(bucket, entry string) (string, error) {
 	// Read from fs metadata only if it exists.
 	defer fs.rwPool.Close(fsMetaPath)
 
-	fsMetaBuf, err := ioutil.ReadAll(rlk.LockedFile)
+	fi, err := rlk.LockedFile.Stat()
```

Comment:
You could add a comment why `Stat()` then `ioutil.ReadAll()` approach is required with `ReadAt()`